modules/apps/data-engine/data-engine-test/src/testFunctional/tests/DEFields.testcase

```diff
@@ -0,0 +1,111 @@
+@component-name = "data-engine"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Data Engine";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			WebContentStructures.tearDownCP();
+		}
+	}
+
+	@description = "LPS-168179 - It's possible to reorder Web Content with Structure of repeatable fields"
+	@priority = 5
+	test ReorderRepeatableFieldsByDragAndDrop {
+		task ("Given a Web Structure that contains repeatable fields") {
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Web Content");
+
+			NavItem.gotoStructures();
+
+			WebContentStructures.addCP(structureName = "WC Structure Title");
+
+			DEBuilder.addField(
+				fieldLabel = "Text field 1",
+				fieldName = "Text",
+				value = "Entry 1");
+
+			Click(
+				key_fieldLabel = "Text field 1",
+				key_fieldName = "Text",
+				locator1 = "FormFields#TEXT_FIELD",
+				value1 = "Text field 1");
+
+			FormViewBuilder.setPlaceholder(fieldValue = "Entry 1");
+
+			Form.gotoAdvancedTab();
+
+			Check.checkToggleSwitch(
+				checkboxName = "Repeatable",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+
+			WaitForElementPresent(locator1 = "Button#PLUS");
+
+			Button.clickSave();
+
+			WaitForPageLoad();
```

Comment:
@mateussandes23 how is this waiting for the duplicate button if this is done before navigating to web content and adding a structure?

Reply:
The same for this one. Will remove either.